src/main/java/com/kryptnostic/datastore/services/EdmService.java

```diff
@@ -570,6 +570,7 @@ public void updateEntityTypeMetadata( UUID entityTypeId, MetadataUpdate update )
             aclKeyReservations.renameReservation( entityTypeId, update.getType().get() );
         }
         entityTypes.executeOnKey( entityTypeId, new UpdateEntityTypeMetadataProcessor( update ) );
+        eventBus.post( new EntityTypeCreatedEvent( getEntityType( entityTypeId ) ) );
```

Comment:
because it saves the updated entity type to elasticsearch. check out SearchService